### PR TITLE
Discoverable also for home-assistant.io

### PR DESCRIPTION
**Pull request by selvakn**
cc: @dhivyada

### CODE REVIEW

File: src/WemoSwitch.cpp [hunk 1]
```diff
@@ -9,7 +9,7 @@ WemoSwitch::WemoSwitch(){
 }
 //WemoSwitch::WemoSwitch(String alexaInvokeName,unsigned int port){
 WemoSwitch::WemoSwitch(String alexaInvokeName, unsigned int port, CallbackFunction oncb, CallbackFunction offcb){
-    uint32_t chipId = ESP.getChipId();
+    uint32_t chipId = ESP.getChipId() + port;
```
Comment (witnessmenow) [inline src/WemoSwitch.cpp]: small comment but I would change the variable name to reflect its the chipIdWithPort

**Comment by witnessmenow** [comment]
Looks good, thanks for the PR. Have you tested that it still works with Alexa after the changes? 

I can test it over the weekend if you haven't got a chance

**Comment by selvakn** [comment]
You are welcome!
I have tested this with alexa and home-assistant.io, for single and multiple switches.

File: src/WemoSwitch.cpp (hunk 1 above)
Comment (selvakn) [inline src/WemoSwitch.cpp]: Fixed!

**Pull request merged**